corehq/apps/app_manager/models.py

```diff
@@ -5311,6 +5369,13 @@ def fetch_file(self, location):
 
         return location, content
 
+    def get_build_langs(self):
+        if self.build_profiles:
+            # return first profile, generated as part of lazy migration
+            return self.build_profiles[self.build_profiles.keys()[0]].langs
```

Comment:
Implied in previous comment: does it matter if the resturn value of this function is non-deterministic?